docs/filter.md

```diff
@@ -94,3 +96,13 @@ Create a "base" subset that only includes internal axioms (alternatively, use `r
       --include-term IAO:0000119 \
       --output results/template-base-filter.owl
 
+Create a "base" subset that axiom annotations related with the two properties are removed:
```

Comment:
Create a "base" subset in which axiom annotations involving IAO:0000117 and IAO:0000119 are removed: